--- apertium-fr-ca-1.0.3/configure.ac.orig	2015-12-29 21:19:52.855328855 +0100
+++ apertium-fr-ca-1.0.3/configure.ac	2015-12-29 21:22:56.165321162 +0100
@@ -32,8 +32,12 @@
                 APERTIUM_VER=31
               ],
               [
-                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0)
+                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0,
+                  [
                   APERTIUM_VER=32
+                  ],
+                  [PKG_CHECK_MODULES(APERTIUM, apertium >= 3.3.0)
+                   APERTIUM_VER=3])
               ])
           ])
       ])
