--- apertium-fr-ca-1.0.2/Makefile.am.orig	2009-03-12 20:49:44.000000000 +0100
+++ apertium-fr-ca-1.0.2/Makefile.am	2010-12-20 17:58:37.094115231 +0100
@@ -91,8 +91,8 @@
 
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
-	$(INSTALL_DATA) $(PREFIX1).mode $(apertium_fr_modesdir)
-	$(INSTALL_DATA) $(PREFIX2).mode $(apertium_fr_modesdir)
-	$(INSTALL_DATA) $(BASENAME).$(PREFIX2).t1x $(BASENAME).$(PREFIX1).t1x $(apertium_fr_cadir)
+	$(INSTALL_DATA) $(PREFIX1).mode $(DESTDIR)$(apertium_fr_modesdir)
+	$(INSTALL_DATA) $(PREFIX2).mode $(DESTDIR)$(apertium_fr_modesdir)
+	$(INSTALL_DATA) $(BASENAME).$(PREFIX2).t1x $(BASENAME).$(PREFIX1).t1x $(DESTDIR)$(apertium_fr_cadir)
 
 CLEANFILES = -rf $(TARGETS) modes $(BASENAME).$(LANG1).dix *.mode
